tests/Integration/Behaviour/Features/Scenario/CartRule/add_cart_rule.feature

```diff
@@ -0,0 +1,81 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s cart_rule
+@reset-database-before-feature
+Feature: Add cart rule
+  PrestaShop allows BO users to create cart rules
+  As a BO user
+  I must be able to create cart rules
+
+  Background:
+    Given shop "shop1" with name "test_shop" exists
+    Given there is a currency named "currency1" with iso code "USD" and exchange rate of 0.92
+    Given there is a currency named "currency2" with iso code "CHF" and exchange rate of 1.25
+    Given currency "currency1" is the default one
+
+  Scenario: Create a cart rule with amount discount
+    When I want to create a cart rule named "Promotion"
+    And I specify its "description" as "Promotion for holidays"
+    And I specify that its active from "2019-01-01 11:05:00"
+    And I specify that its active until "2019-12-01 00:00:00"
+    And I specify that its "quantity" is "10"
+    And I specify that its "quantity per user" is "1"
+    And I specify that its "priority" is "2"
+    And I specify that partial use is disabled for it
+    And I specify its status as enabled
+    And I specify that it should not be highlighted in cart
+    And I specify its "code" as "PROMO_2019"
+    And its minimum purchase amount in currency "CHF" is "10"
+    And its minimum purchase amount is tax excluded
+    And its minimum purchase amount is shipping included
+    And it gives free shipping
+    And it gives a reduction amount of "15" in currency "USD" which is tax included and applies to order without shipping
+    When I successfully save it
```

Comment:
Can you "unsuccessfully" save it? :grin: I think "successfully" is not needed here.